.github/workflows/deploy.yml

```diff
@@ -7,4 +7,17 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Dump event
-        run: jq . $GITHUB_EVENT_PATH
\ No newline at end of file
+        run: jq . $GITHUB_EVENT_PATH
+
+      - name: Make ssh key
+        run: mkdir -p ~/.ssh && touch ~/.ssh/testing.pem && echo "${{secrets.TESTING_PEM}}" > ~/.ssh/testing.pem && chmod 400 ~/.ssh/testing.pem
+          
+      - uses: actions/checkout@v2
+      - name: Install ansible roles
+        run: ansible-galaxy install -r /home/runner/work/multinet-deploy/multinet-deploy/ansible/requirements.yml
+
+      - name: Add server to known hosts
+        run:  touch ~/.ssh/known_hosts && ssh-keyscan 18.217.253.44 >> ~/.ssh/known_hosts
```

Comment:
Is there a way to abstract out this IP address as a variable? Since it appears twice, and can change over the lifetime of the deployment, it would be good to have it defined in one spot.

Reply:
We could do it as a secret?

Reply:
That's probably too far. But I think I just read that you can set an environment for a job?

If we convert this problem to an issue in this repo, I think that's good enough for now.

Reply:
Opened #23